corehq/apps/export/views/list.py

```diff
@@ -91,7 +91,13 @@ def __init__(self, request, domain):
         self.domain = domain
         self.permissions = ExportsPermissionsManager(self.form_or_case, domain, request.couch_user)
 
+    def _priv_check(self):
+        return True
```

Comment:
This helper class seems like an odd place for a permissions check - I'd expect it to happen in the view function.

Reply:
Yeah, I debated what to do with this logic, ultimately moved it here for the sake of DRYer code - if it happens in the view function I need to duplicate the logic in the `get_exports_page` view.